shared/markdown/parser.pegjs

```diff
@@ -118,25 +115,15 @@ Strike
  = StrikeMarker !WhiteSpace children:StrikeInline StrikeMarker !(StrikeMarker / NormalChar) { return {type: 'strike', children: flatten(children)} }
 
 // children grammar adapted from username regexp in libkb/checkers.go.
-Mention
- = MentionMarker children:([a-zA-Z0-9][a-zA-Z0-9_]*) MentionMarker service:ValidMentionService { return {type: 'mention', children: flatten(children), service: service.toLowerCase()} }
-
-// Same as Mention above, but is just returns text
-// Useful if you don't want a mention in certain contexts (like in a code block)
-MentionlessMention
- = MentionMarker children:([a-zA-Z0-9][a-zA-Z0-9_]*) MentionMarker service:ValidMentionService { return prefix('@', children) }
-
-InCodeBlock
- = children:(MentionlessMention / (!Ticks3 .))+ {return children }
+Mention = MentionMarker children:([a-zA-Z0-9]+[a-zA-Z0-9_]?)+ & {
```

Comment:
Ugh, I didn't mean to re-introduce a bug here. But https://github.com/keybase/client/pull/10108/files will fix it anyway.

Reply:
Nevermind, I'm confusing myself, this one is actually a good regexp.